src/applications/ivc-champva/10-7959a/chapters/healthInsuranceInformation.js

```diff
@@ -0,0 +1,231 @@
+    summaryTitle: 'Beneficiary’s health insurance review',
+  },
+};
+
+export const insuranceStatusSchema = {
+  uiSchema: {
+    ...titleUI(({ formData }) => {
+      return `${nameWording(
+        formData,
+        true,
+        true,
+        true,
+      )} health insurance status`;
+    }),
+    hasOhi: {
+      ...yesNoUI({
+        updateUiSchema: formData => {
+          return {
+            'ui:title': `${
+              formData?.certifierRole === 'applicant' ? 'Do' : 'Does'
+            } ${nameWording(
+              formData,
+              false,
+              false,
+              true,
+            )} have health insurance coverage that isn’t through CHAMPVA?`,
+            'ui:options': {
+              hint: hintText,
+            },
+          };
+        },
+      }),
+    },
+  },
+  schema: {
+    type: 'object',
+    required: ['hasOhi'],
+    properties: {
+      titleSchema,
+      hasOhi: yesNoSchema,
+    },
+  },
+};
+
+const policyPage = {
+  uiSchema: {
+    ...arrayBuilderItemFirstPageTitleUI({
+      title: 'Policy information',
+      nounSingular: options.nounSingular,
+    }),
+    name: textUI('Name of insurance provider'),
+    policyNum: textUI('Policy number'),
+    providerPhone: phoneUI('Insurance provider phone number'),
+  },
+  schema: {
+    type: 'object',
+    properties: {
+      name: textSchema,
+      policyNum: textSchema,
+      providerPhone: phoneSchema,
+    },
+    required: ['name'],
+  },
+};
+
+const insuranceProviderPage = {
+  uiSchema: {
+    ...arrayBuilderItemSubsequentPageTitleUI(
+      ({ formData }) => `${get('name', formData)} insurance type`,
+    ),
+    type: {
+      ...radioUI({
+        title:
+          'What type of insurance does the beneficiary have through this provider?',
+        labels: radioLabels,
+        errorMessages: {
+          required: 'Please select an option',
+        },
+      }),
+    },
+    otherType: {
+      'ui:title': `Since your insurance type was not listed, please describe it here`,
+      'ui:webComponentField': VaTextInputField,
+      'ui:options': {
+        expandUnder: 'type',
+        expandUnderCondition: 'other',
+        expandedContentFocus: true,
+      },
+      'ui:errorMessages': {
+        required: `Please enter your insurance type`,
+      },
+    },
+    'ui:options': {
+      updateSchema: (formData, formSchema) => {
+        if (formSchema.properties.otherType['ui:collapsed']) {
+          return { ...formSchema, required: ['type'] };
+        }
+        return {
+          ...formSchema,
+          required: ['type', 'otherType'],
+        };
+      },
+    },
+  },
+  schema: {
+    type: 'object',
+    properties: {
+      titleSchema,
+      type: radioSchema(Object.keys(radioLabels)),
+      otherType: textSchema,
+    },
+  },
+};
+
+const summaryPage = {
+  uiSchema: {
+    'view:hasPolicies': arrayBuilderYesNoUI(
+      options,
+      yesNoContent,
+      yesNoContent,
```

Comment:
Have to pass this object twice - these two properties control different conditional displays, but in this case we want the same custom text to show up in both cases.